packages/core/integration-tests/ready-yet-reporter.js

```diff
@@ -0,0 +1,56 @@
+// my-reporter.js
+const mocha = require('mocha');
+const path = require('path');
+const fs = require('fs');
+const {execSync} = require('child_process');
+
+function importantData(test, status) {
+  let fileName = path.basename(test.file, '.js');
+  return {
+    fileName,
+    link: `https://github.com/padmaia/parcel/tree/pull-out-integration-tests/packages/core/integration-tests/test/${fileName}.js`,
+    title: test.title,
+    status
+  };
+}
+
+function ReadyYetReporter(runner) {
+  mocha.reporters.Base.call(this, runner);
+  let tests = [];
+  let numPassing = 0;
+
+  runner.on('pending', function(test) {
+    tests.push(importantData(test, 'pending'));
+  });
+
+  runner.on('pass', function(test) {
+    numPassing++;
+    tests.push(importantData(test, 'passing'));
+  });
+
+  runner.on('fail', function(test, err) {
+    tests.push(importantData(test, 'failing'));
+  });
+
+  runner.on('end', function() {
+    let ratio = `${numPassing}/${tests.length}`;
+    let commitHash = execSync('git rev-parse HEAD').toString(); // Get rid of newline
+    commitHash = commitHash.substring(0, commitHash.length - 1);
+    let commitDate = execSync(
+      `git show -s --format=%ci ${commitHash}`
+    ).toString();
+    commitDate = commitDate.substring(0, commitDate.lastIndexOf(' ')); // Get rid of timezone
+    let testHistory = JSON.parse(
+      fs.readFileSync('data/testHistory.json', 'utf8')
+    );
+    testHistory.push(`${commitHash}\t${commitDate}\t${ratio}`);
+    fs.writeFileSync(
+      'data/testHistory.json',
+      JSON.stringify(testHistory),
+      'utf8'
+    );
```

Comment:
@devongovett what's the best way to persist the test history file? I was thinking the simplest way would have the CI server commit and push based on test results for every push to the v2 branch.

Reply:
Could maybe even deploy the app after too, but not sure if you want to give the CI server that much responsibility.

Reply:
We could probably just run `now` in there to deploy it from CI?